docs/tutorials/projects/light-up.md

```diff
@@ -84,12 +79,83 @@ Navigate to the **Control** tab where you can see your camera working.
 
 {{< gif webm_src="../../img/light-up/control-panel.webm" mp4_src="../../img/light-up/control-panel.mp4" alt= "Opening the camera panel on the control tab, toggling the video feed on, and watching as a person wearing headphones waves at the camera." >}}
 
+## Configure your services
+
+This tutorial uses pre-trained ML packages.
+If you want to train your own, you can [train a model](/manage/ml/train-model/).
+
+To use the provided Machine Learning model, copy the <file>[effdet0.tflite](https://github.com/viam-labs/devrel-demos/raw/main/Light%20up%20bot/effdet0.tflite)</file> file and the <file>[labels.txt](https://github.com/viam-labs/devrel-demos/raw/main/Light%20up%20bot/labels.txt)</file> to your project directory.
+
+{{< tabs >}}
```

Comment:
There's an error somewhere here (which you can see in the logs of the failed testes):
<img width="794" alt="Screenshot 2023-07-10 at 20 22 15" src="https://github.com/viamrobotics/docs/assets/5212232/0916a0c0-85bc-4267-ab1e-0475ed38b529">

I think you're not closing the tabs :)

Reply:
thanks so much!! removed the tabs